manifests/deployment.yml

```diff
@@ -0,0 +1,88 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: prometheus-status-cake-exporter
+  labels:
+    app: prometheus-status-cake-exporter
+spec:
+  ports:
+  - name: metrics
+    port: 8000
+    protocol: TCP
+  selector:
+    app: prometheus-status-cake-exporter
+
+---
+apiVersion: v1
+kind: Secret

```

Comment:
In my experience it's better to create secrets using the Azure DevOps task, I found with the das-grafana pipeline that the Azure DevOps variable needs to be base64 encoded if you use a manifest file.  Example [here](https://github.com/SkillsFundingAgency/das-grafana/blob/6a110577323f72bf38e4194b73593c4e76f3e602/pipeline-templates/job/deploy-grafana.yml#L88)

Reply:
The deployment error due to 'illegal base64 data' was avoided when stringData was used instead of data https://stackoverflow.com/a/60773270

The pod's secrets are configured correctly when viewing the pod

Reply:
Azure DevOps task used